src/hooks/use-site-details.tsx

```
@@ -101,8 +101,6 @@ function useDeleteSite() {
 				const newSites = await getIpcApi().deleteSite( siteId, removeLocal );
 				await allSiteRemovePromises;
 				return newSites;
-			} catch ( error ) {
-				throw new Error( 'Failed to delete local files' );
```

We let the exception bubble up so we can display the original error message.